drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java

```diff
@@ -80,6 +80,9 @@ private void init() {
         sDraweeControllerBuilderSupplier,
         "SimpleDraweeView was not initialized!");
     mSimpleDraweeControllerBuilder = sDraweeControllerBuilderSupplier.get();
+    if (mImageUri != null && !mImageUri.isEmpty()){
```

Comment:
This is the place to extract the uri from `R.styleable.SimpleDraweeView_imageUri`